app/controllers/jurisdictions_controller.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+# JurisdictionsController: handles all subject actions
+class JurisdictionsController < ApplicationController
+  before_action :authenticate_user!
+
+  # Get jurisdiction ids and paths of viewable jurisdictions
+  def jurisdiction_paths
+    render status: 403 unless current_user.can_create_patient? || current_user.can_edit_patient? || current_user.can_view_public_health_dashboard?
+
+    render json: { jurisdictionPaths: Hash[current_user.jurisdiction.subtree.pluck(:id, :path).map { |id, path| [id, path] }] }
+  end
+
+  # Get list of assigned users unique to jurisdiction
+  def assigned_users
+    render status: 403 unless current_user.can_create_patient? || current_user.can_edit_patient? || current_user.can_view_public_health_dashboard?
+
+    permitted_params = params.permit(:jurisdiction_id, :scope, :workflow, :tab)
+
+    # Validate jurisdiction_id param
+    jurisdiction_id = permitted_params[:jurisdiction_id].to_i
+    render status: 400 unless current_user.jurisdiction.subtree_ids.include?(jurisdiction_id)
+    jurisdiction = Jurisdiction.find(jurisdiction_id)
+
+    # Validate scope param
+    scope = permitted_params[:scope].to_sym
+    render status: 400 unless %i[all exact].include?(scope)
+
+    # Validate workflow param
+    workflow = permitted_params[:workflow].to_sym unless permitted_params[:workflow].nil?
+    render status: 400 if workflow && !%i[exposure isolation].include?(workflow)
+
+    # Validate tab param
+    tab = permitted_params[:tab].to_sym unless permitted_params[:tab].nil?
+    render status: 400 if tab && workflow.nil?
+    render status: 400 if workflow == 'exposure' && !%i[all symptomatic non_reporting asymptomatic pui closed transferred_in].include?(tab)
+    render status: 400 if workflow == 'isolation' && !%i[all requiring_review non_reporting reporting closed transferred_in].include?(tab)
+
+    # Start by getting all or immediate patients from jurisdiction
+    patients = scope == :all ? jurisdiction.all_patients : jurisdiction.immediate_patients
+
+    # Filter by workflow and tab
+    if workflow == :exposure
+      patients = patients.where(isolation: false, purged: false)
+      patients = patients.exposure_symptomatic if tab == :symptomatic
+      patients = patients.exposure_non_reporting if tab == :non_reporting
+      patients = patients.exposure_asymptomatic if tab == :asymptomatic
+      patients = patients.exposure_under_investigation if tab == :pui
+    end
+
+    if workflow == :isolation
+      patients = patients.where(isolation: true, purged: false)
+      patients = patients.isolation_requiring_review if tab == :requiring_review
+      patients = patients.isolation_non_reporting if tab == :non_reporting
+      patients = patients.isolation_reporting if tab == :reporting
+    end
+
+    patients = patients.monitoring_closed_without_purged if tab == :closed
```

Comment:
so those should already be scoped by workflow in these lines (based on which workflow):

https://github.com/SaraAlert/SaraAlert/blob/SARAALERT-483/app/controllers/jurisdictions_controller.rb#L44

https://github.com/SaraAlert/SaraAlert/blob/SARAALERT-483/app/controllers/jurisdictions_controller.rb#L52

the reason why the `closed` one is outside of those `if` statements is because the same `monitoring_closed_without_purged` scope is applied no matter the workflow